subprojects/packagefiles/libcap/meson.build

```diff
@@ -0,0 +1,49 @@
+project('libcap', 'c',
+  version: '2.76',
+  license: 'BSD-3-Clause OR GPL-2.0-only',
+  default_options: [
+    'c_std=gnu11',
+    'buildtype=release']
+  )
+
+incdir = include_directories('libcap/include')
+
+cap_names_list_h = custom_target('cap_names.list.h',
+  output: 'cap_names.list.h',
+  input: 'libcap/include/uapi/linux/capability.h',
+  command: [
+    files('generator.sh'),
```

Comment:
You can't call this `generator.sh` because that name has special significance in WrapDB.